core/api/src/app/payments/send-lightning.ts

```diff
@@ -411,6 +411,26 @@ const executePaymentViaIntraledger = async <
   })
   if (limitCheck instanceof Error) return limitCheck
 
+  const recipientUser = await UsersRepository().findById(recipientUserId)
+  if (recipientUser instanceof Error) return recipientUser
+
+  const sendNotificationRecipientPartialArgs = {
+    accountId: recipientAccount.id,
+    walletId: recipientWalletDescriptor.id,
+    userId: recipientUser.id,
+    level: recipientAccount.level,
+  }
+
+  const senderUser = await UsersRepository().findById(senderAccount.kratosUserId)
+  if (senderUser instanceof Error) return senderUser
+
+  const sendNotificationSenderPartialArgs = {
```

Comment:
not a blocker but.. this is the recipient param... sendNotificationSenderPartialArgs is a bit confusing

Reply:
We've overloaded the word `recipient`. These are the "message recipient" params for the "transaction sender" in an intraledger context (original code [here](https://github.com/GaloyMoney/galoy/pull/4039/files#diff-baced5c5a6f65916e2408be30a1c43c54bfbe5c4399f0fadf81144c04467c41dL565)).

We also have "transaction recipient" params just above this that came (original code [here](https://github.com/GaloyMoney/galoy/pull/4039/files#diff-baced5c5a6f65916e2408be30a1c43c54bfbe5c4399f0fadf81144c04467c41dL547))

I can try to make the names a bit cleaner

Reply:
Ah I can change this. It's confusing because I removed an abstraction that was using this differently

Reply:
Can let me know if this is any better https://github.com/GaloyMoney/galoy/pull/4039/commits/09f39ca8cbee2375d7611f3990cbfea6fd3fb89c